### DISCUSSION TITLE
blog/blend_v4/

### DISCUSSION THREAD

**Discussion by giscus** [Ast3risk-ops/website#2] [Giscus]
# blog/blend_v4/

Welcome, weary traveler.

https://asterisk.lol/blog/blend_v4/

**Comment by liluzibird** [comment]
Yoo cool, there's giscus integration in this too. I'll keep following your feed as always. Glad to see the website up and running. 

Edit: the messages as I'm loading it made me feel like I was playing Celeste or something

**Reply by Ast3risk-ops** [reply]
Not officially, I had to add the Giscus script to a comments partial.

**Reply by liluzibird** [reply]
Seems cool. If I ever want to set up Blowfish, I'll probably check this repo out.

**Reply by Ast3risk-ops** [reply]
Honestly the best part is probably that it's so easy to add icons and it looks staggering while remaining fast. It also has more shortcodes :P

**Reply by liluzibird** [reply]
Oo I like that you added your gpg key there. I didn't really know where to put mine so I put it in [contact](https://liluzibird.github.io/contact/). Yo, I'll definitely put my steam and stuff on my page as well.

**Reply by Ast3risk-ops** [reply]
You can link yours from github if you want it as a social: `https://github.com/<USERNAME>.gpg`

**Reply by liluzibird** [reply]
The gpg key I use to make commits? My gpg key for email is different.

**Reply by Ast3risk-ops** [reply]
Oh :P Either way gl with your site, you should get a domain for it.

**Reply by liluzibird** [reply]
I was thinking about it, and I don't know. I don't really want to pay for a domain when I'm kind of unsure about my GitHub account name at the moment. liluzibert is taken, and I kind of don't want to do a variation of that name.

**Reply by Ast3risk-ops** [reply]
Feel free to star the repo tho ⭐
